src/models/azure-key-vault-secret-status/index.ts

```diff
@@ -1,13 +1,23 @@
 import * as PropTypes from 'prop-types';
 
 export interface AzureKeyVaultSecretStatusModel {
-  replicaName?: string;
+  replicaName: string;
+  replicaCreated: Date;
```

Comment:
is `replicaCreated` always supplied by the API or is it actually optional?

Reply:
Allways